istioctl/cmd/kubeinject.go

```diff
@@ -153,12 +290,34 @@ func getInjectConfigFromConfigMap(kubeconfig, revision string) (inject.Templates
 	return injectConfig.Templates, nil
 }
 
+func setUpExternalInjector(kubeconfig, revision string) (*ExternalInjector, error) {
+	e := &ExternalInjector{nil, nil}
+	client, err := kube.NewExtendedClient(kube.BuildClientCmd(kubeconfig, configContext), "")
+	if err != nil {
+		return e, err
+	}
+	if whcName == defaultInjectWebhookConfigName && revision != "" {
+		whcName = fmt.Sprintf("%s-%s", defaultInjectWebhookConfigName, revision)
+	}
+	whc, err := client.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.TODO(),
```

Comment:
Maybe we should instead query for `istio.io/rev=<revision>`. This is a bit rigid and would break if users have a different name. We do the same in istiod today.

Reply:
done